test/integration/metrics_health_test.lua

```diff
@@ -35,6 +35,14 @@ g.after_all(function()
     fio.rmtree(g.cluster.datadir)
 end)
 
+g.before_each(function()
+    local main_server = g.cluster:server('main')
+    main_server:exec(function()
+        local metrics = require('cartridge.roles.metrics')
+        metrics.set_custom_is_health_handler(nil)
```

Comment:
Maybe it's better to make a single cleanup in `g.after_test('test_metrics_custom_is_health_handler', function)`